src/left.jl

```diff
@@ -50,6 +50,7 @@ end
 function mul!(x::AbstractMatrix, y::AdjointAbsVecOrMat, A::LinearMap, α::Number, β::Number)
     check_dim_mul(x, y, A)
     _unsafe_mul!(x', conj(α)*A', y', true, conj(β))
+    # why not?: _unsafe_mul!(x', A', y', conj(α), conj(β))
```

Comment:
This is because 5-arg `mul!` always interprets (and performs) the multiplication by `α` from the right. But when we transpose/adjoint the whole product, the order should be formally reversed, and for non-commutative types that order matters. But this is of no harm, actually, for `RealOrComplex`: in this case, `conj(α)*A'` creates a `ScaledMap`

https://github.com/Jutho/LinearMaps.jl/blob/b38c1ac1900d4751237bbef62927477017f4f7d0/src/scaledmap.jl#L30

and `mul!` for `ScaledMap`s does exactly what you propose:

https://github.com/Jutho/LinearMaps.jl/blob/b38c1ac1900d4751237bbef62927477017f4f7d0/src/scaledmap.jl#L46-L55